src/components/SidePanel/SidePanel.jsx

```diff
@@ -48,18 +49,21 @@ export default function SidePanel({ request, response, content, query, clearSele
             </button>
           );
         })}
+        <button type="button" className={closeClasses} onClick={clearSelectedRequest}>
+          &times;
+        </button>
       </div>
 
       <div className={styles.tabContent}>
         {activeTab === RESPONSE ? (
           <>
+            <h3>Headers</h3>
+            <HeadersTable headers={response.headers} />
```

Comment:
Why move the response headers above the response but leave the request headers below the request?

I've flipped the order back and forth a couple of times. I was leaning towards keeping the headers at the bottom since, during development, it's more likely that we inspect the gql operations and the associated response data than the headers. I'm open to putting the headers up top if you think it will be beneficial, but let's also move the request headers to the top of their tab.

Reply:
The headers, to me are pretty important to see a lot of basic information about the request before viewing the data, especially before debugging. For example, getting the bearer token when auth errors occur, or just making sure you're going to the correct URL. It'd be pretty frustrating to have to scroll through 1000 objects in response to get header data.

Reply:
Alright, I'm on board. Let's move the request headers up above the printed query to match the response tab.